docs/framework/wpf/getting-started/index.md

```diff
@@ -16,7 +16,7 @@ Windows Presentation Foundation (WPF) is a UI framework that creates desktop cli
 |-|-|  
 |I want to jump right in…|[Walkthrough: My first WPF desktop application](walkthrough-my-first-wpf-desktop-application.md)|  
 |How do I design the application UI?|[Designing XAML in Visual Studio](/visualstudio/designers/designing-xaml-in-visual-studio)|  
-|New to .NET?|[Overview of the .NET Framework](../../get-started/overview.md)<br /><br /> [.NET Framework Application Essentials](../../../standard/application-essentials.md)<br /><br /> [Getting Started with Visual C# and Visual Basic](/visualstudio/ide/getting-started-with-visual-csharp-and-visual-basic)|  
+|New to .NET?|[Overview of the .NET Framework](../../get-started/overview.md)<br /><br /> [.NET Framework Application Essentials](../../../standard/application-essentials.md)<br /><br /> [Getting Started with Visual C# and Visual Basic](/visualstudio/ide/quickstart-visual-basic-console)|  
```

Comment:
Since this link not under the csharp or visual-basic. I think both should be added. Thoughts ?